pubtools/_ami/tasks/push.py

```diff
@@ -218,21 +218,23 @@ def upload(self, push_item):
         else:
             accounts = list(_accounts["default"].values())
 
-        publish_meta = AWSPublishingMetadata(
-            image_path=file_path,
-            image_name=name,
-            snapshot_name=name,
-            container=container,
-            description=push_item.description,
-            arch=push_item.release.arch,
-            virt_type=push_item.virtualization,
-            root_device_name=push_item.root_device,
-            volume_type=push_item.volume,
-            billing_products=push_item.billing_codes.codes,
-            accounts=accounts,
-            sriov_net_support=push_item.sriov_net_support,
-            ena_support=push_item.ena_support or False,
-        )
+        publishing_meta_kwargs = {
+            "image_path": file_path,
+            "image_name": name,
+            "snapshot_name": name,
+            "container": container,
+            "description": push_item.description,
+            "arch": push_item.release.arch,
+            "virt_type": push_item.virtualization,
+            "root_device_name": push_item.root_device,
+            "volume_type": push_item.volume,
+            "billing_products": push_item.billing_codes.codes,
+            "accounts": accounts,
+            "sriov_net_support": push_item.sriov_net_support,
+            "ena_support": push_item.ena_support or False,
+        }
+        LOG.info(publishing_meta_kwargs)
```

Comment:
I've got a couple of issues with this:

- If we're dumping some python objects without further context in the message, could you please make it debug instead of info?  The way it is now, the message is not really understandable to end-users.  Remember we do have the ability to set log levels in Pub settings now, so this wouldn't lock us out of getting the logs when a problem occurs.
- I believe it should be `LOG.debug("%s", publishing_meta_kwargs)`.  I checked the python logging docs and there's no mention of what the behavior is if the 'msg' argument is not a string. Seems risky to assume it means to format the argument as a string.